.github/actions/image-scan/action.yml

```diff
@@ -0,0 +1,40 @@
+name: 'Scan docker image'
+description: Scan docker image with trivy and dockle
+inputs:
+  image:
+    description: 'Image file to scan'
+    required: true
+runs:
+  using: "composite"
+  steps:
+    - name: Run Dockle
+      uses: erzz/dockle-action@v1
+      with:
+        image: ${REGISTRY}/backbase/${GITHUB_REPOSITORY##*/}:${{ inputs.version }}
```

Comment:
input seems to be `${REGISTRY}/backbase/${GITHUB_REPOSITORY##*/}:${{ inputs.version }}` -